### PR TITLE
[CIMB Singapore] Update domains

### PR DESCRIPTION
**Pull request by youdly**
https://github.com/EFForg/https-everywhere/pull/14035

### CODE REVIEW

File: src/chrome/content/rules/CIMB-Singapore.xml [hunk 1]
```diff
@@ -1,45 +1,21 @@
...
-		- (www.)cimbbank.com.sg		(Expired cert 12/04/15 09:29)
-
+	For other CIMB coverage, see CIMB.com.xml
 -->
-
 <ruleset name="CIMB Singapore">
 
-	<target host=    "cimbclicks.com.sg" />
+	<target host="cimbbank.com.sg" />
+	<target host="www.cimbbank.com.sg" />
```
Comment (J0WI) [inline src/chrome/content/rules/CIMB-Singapore.xml]: `secure.eforms.cimbbank.com.sg` (requires custom test url)

File: src/chrome/content/rules/CIMB-Singapore.xml [hunk 2]
```diff
@@ -1,45 +1,21 @@
...
+
+	<target host="cimbclicks.com.sg" />
 	<target host="www.cimbclicks.com.sg" />
```
Comment (J0WI) [inline src/chrome/content/rules/CIMB-Singapore.xml]: `m.cimbclicks.com.sg` (m)

File: src/chrome/content/rules/CIMB-Singapore.xml [hunk 3]
```diff
@@ -1,45 +1,21 @@
...
-	<target host=    "bizchannel.cimb.com.sg" />
+
```
Comment (J0WI) [inline src/chrome/content/rules/CIMB-Singapore.xml]: Do `cimb.com.sg` and `www.cimb.com.sg` work for you?

File: src/chrome/content/rules/CIMB-Singapore.xml (hunk 1 above)
Comment (J0WI) [reply src/chrome/content/rules/CIMB-Singapore.xml]: ...and an exclusion on `/$`

File: src/chrome/content/rules/CIMB-Singapore.xml (hunk 3 above)
Comment (youdly) [reply src/chrome/content/rules/CIMB-Singapore.xml]: No, I've added them in the ruleset's ruleset.

File: src/chrome/content/rules/CIMB-Singapore.xml (hunk 1 above)
Comment (youdly) [reply src/chrome/content/rules/CIMB-Singapore.xml]: Added exclusion

**Pull request merged**